<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookDialog</class>
 <widget class="QDialog" name="BookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>İsim</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_title"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Yazar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_author"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Yayınevi</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_publisher"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Sayfa Sayısı</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spinBox_pageNumber">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Stok</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spinBox_stock">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Kategori</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="comboBox_category">
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="maxVisibleItems">
      <number>25</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Basım Yılı</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="spinBox_publishingYear">
     <property name="minimum">
      <number>1900</number>
     </property>
     <property name="maximum">
      <number>2100</number>
     </property>
     <property name="value">
      <number>2005</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Demirbaş No</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spinBox_fixture">
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Cilt No</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QSpinBox" name="spinBox_vol">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>lineEdit_author</tabstop>
  <tabstop>lineEdit_publisher</tabstop>
  <tabstop>spinBox_pageNumber</tabstop>
  <tabstop>spinBox_stock</tabstop>
  <tabstop>comboBox_category</tabstop>
  <tabstop>spinBox_publishingYear</tabstop>
  <tabstop>spinBox_fixture</tabstop>
  <tabstop>spinBox_vol</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
